wgsl/index.bs

```diff
@@ -1129,6 +1135,14 @@ type, function decorations and storage class.
     <tr><td>local_invocation_id<td>vec3&ltu32&gt<td>Compute Input
     <tr><td>global_invocation_id<td>vec3&ltu32&gt<td>Compute Input
     <tr><td>local_invocation_idx<td>u32<td>Compute Input
+    <tr><td>subgroup_size<td>u32<td>Compute Input<br>
+                                    Exists if "subgroup-operations"<br>
+                                    is both supported by the device/user<br>
```

Comment:
we don't need to specify this as "both", since the second one requires the first one. We should just talk about what's enabled on the device

Reply:
I took that from `6.3. Texture Formats` in host spec, does it look good now?